src/main/java/com/unzer/payment/Unzer.java

```diff
@@ -57,1079 +51,1088 @@
  * @see UnzerRestCommunication for details of the http-layer abstraction.
  */
 public class Unzer {
-	private String privateKey;
-	private String endPoint;
-	private transient PaymentService paymentService;
-	private transient MarketplacePaymentService marketplacePaymentService;
-	private transient PaypageService paypageService;
-	private transient LinkpayService linkpayService;
-	private transient WebhookService webhookService;
-
-	public Unzer(String privateKey) {
-		this(new HttpClientBasedRestCommunication(null), privateKey, null);
-	}
-	public Unzer(String privateKey, Locale locale) {
-		this(new HttpClientBasedRestCommunication(locale), privateKey, null);
-	}
-	public Unzer(String privateKey, Locale locale, String endPoint) {
-		this(new HttpClientBasedRestCommunication(locale), privateKey, endPoint);
-	}
-
-	/**
-	 * Creates an instance of the {@code Unzer}-facade.
-	 * @param restCommunication - an appropriate implementation of {@code UnzerRestCommunication}. If you are fine with apache's httpCLient you might choose {@code HttpClientBasedRestCommunication}.
-	 * @param privateKey - your private key as generated within the unzer Intelligence Platform (hIP)
-	 */
-	public Unzer(UnzerRestCommunication restCommunication, String privateKey) {
-		super();
-		this.privateKey = privateKey;
-		this.endPoint = null;
-		this.paymentService = new PaymentService(this, restCommunication);
-		this.marketplacePaymentService = new MarketplacePaymentService(this, restCommunication);
-		this.paypageService = new PaypageService(this, restCommunication);
-		this.linkpayService = new LinkpayService(this, restCommunication);
-		this.webhookService = new WebhookService(this, restCommunication);
-	}
-
-	/**
-	 * Creates an instance of the {@code Unzer}-facade.
-	 * @param restCommunication - an appropriate implementation of {@code UnzerRestCommunication}. If you are fine with apache's httpCLient you might choose {@code HttpClientBasedRestCommunication}.
-	 * @param privateKey - your private key as generated within the unzer Intelligence Platform (hIP)
-	 * @param endPoint - the endPoint for the outgoing connection, in case of null, the value of unzer.properties will be considered
-	 */
-	public Unzer(UnzerRestCommunication restCommunication, String privateKey, String endPoint) {
```

Comment:
This constructor does not exist anymore, right. Could you explain again why it was removed?

Reply:
Do you think there might be some changed behavior for a merchant that might have used this methods so far?